django/db/migrations/loader.py

```diff
@@ -280,6 +280,10 @@ def check_consistent_history(self, connection):
                 continue
             for parent in self.graph.node_map[migration].parents:
                 if parent not in applied:
+                    # Maybe it's a replacing migration? Fixes #27004.
```

Comment:
```
# Skip unapplied squashed migrations that have all of their `replaces` applied.
```
(In general, we don't include ticket numbers in code except for all bu the most obscure issues that can't be described in comments.)